pkg/cli/cmd/radinit/recipe_test.go

```diff
@@ -17,243 +17,110 @@ limitations under the License.
-		},
-		{
-			"Invalid Path #1",
-			"recipes////local-dev/rediscaches",
-			"",
-		},
-		{
-			"Invalid Path #2",
-			"recipes/local-dev////rediscaches",
-			"",
-		},
-		{
-			"Path With Extra Path Argument",
-			"recipes/local-dev/rediscaches/testing",
-			"",
-		},
-	}
-
-	for _, tt := range tests {
-		t.Run(tt.name, func(t *testing.T) {
-			resourceType := getResourceTypeFromPath(tt.repo)
-			require.Equal(t, tt.expected, resourceType)
-		})
-	}
-}
-
-func Test_getPortableResourceType(t *testing.T) {
-	tests := []struct {
-		name         string
-		resourceType string
-		want         string
+		name string
+		want []DevRecipe
 	}{
 		{
-			"Dapr PubSub Portable Resource",
-			"pubsubbrokers",
-			"Applications.Dapr/pubSubBrokers",
-		},
-		{
-			"Dapr Secret Store Portable Resource",
-			"secretstores",
-			"Applications.Dapr/secretStores",
-		},
-		{
-			"Dapr State Store Portable Resource",
-			"statestores",
-			"Applications.Dapr/stateStores",
-		},
-		{
-			"Rabbit MQ Portable Resource",
-			"rabbitmqqueues",
-			"Applications.Messaging/rabbitMQQueues",
-		},
-		{
-			"Redis Cache Portable Resource",
-			"rediscaches",
-			"Applications.Datastores/redisCaches",
-		},
-		{
-			"Mongo Database Portable Resource",
-			"mongodatabases",
-			"Applications.Datastores/mongoDatabases",
-		},
-		{
-			"SQL Database Portable Resource",
-			"sqldatabases",
-			"Applications.Datastores/sqlDatabases",
-		},
-		{
-			"Invalid Portable Resource",
-			"unsupported",
-			"",
+			"Valid Dev Recipes",
+			[]DevRecipe{
+				*NewDevRecipe(
+					"mongodatabases",
+					ds_ctrl.MongoDatabasesResourceType,
+					RecipeRepositoryPrefix+"mongodatabases",
+				),
+				*NewDevRecipe(
+					"rediscaches",
+					ds_ctrl.RedisCachesResourceType,
+					RecipeRepositoryPrefix+"rediscaches",
+				),
+				*NewDevRecipe(
+					"sqldatabases",
+					ds_ctrl.SqlDatabasesResourceType,
+					RecipeRepositoryPrefix+"sqldatabases",
+				),
+				*NewDevRecipe(
+					"rabbitmqqueues",
+					msg_ctrl.RabbitMQQueuesResourceType,
+					RecipeRepositoryPrefix+"rabbitmqqueues",
+				),
+				*NewDevRecipe(
+					"pubsubbrokers",
+					dapr_ctrl.DaprPubSubBrokersResourceType,
+					RecipeRepositoryPrefix+"pubsubbrokers",
+				),
+				*NewDevRecipe(
+					"secretstores",
+					dapr_ctrl.DaprSecretStoresResourceType,
+					RecipeRepositoryPrefix+"secretstores",
+				),
+				*NewDevRecipe(
+					"statestores",
+					dapr_ctrl.DaprStateStoresResourceType,
+					RecipeRepositoryPrefix+"statestores",
+				),
+				*NewDevRecipe(
+					"extenders",
+					ext_ctrl.ResourceTypeName,
+					RecipeRepositoryPrefix+"extenders",
+				),
+			},
 		},
 	}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			if got := getPortableResourceType(tt.resourceType); got != tt.want {
-				t.Errorf("getPortableResourceType() = %v, want %v", got, tt.want)
+			if got := availableDevRecipes(); !reflect.DeepEqual(got, tt.want) {
+				t.Errorf("availableDevRecipes() = %v, want %v", got, tt.want)
```

Comment:
I don't personally find this test kind of valuable, it's testing that the hardcoded data in `recipe.go` is the same as the hardcoded data in `recipe_test.go.